flipperdevices/flipperzero-firmware
# [FL-2548] Infrared: show error if no SD card/databases

# What's new

- Show a comprehensive error if there is no SD card or databases installed upon Universal Remote use.

# Verification 

1. Eject SD card
2. Run Infrared application
3. Go to `Universal Remotes -> TVs`
4. An error message should be shown and yellow LED should be on.
5. Insert the SD card back
6. Repeat steps 2-3. There should be no error.

# Checklist (For Reviewer)
